litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/PanelAndGroup/PanelGroup.tsx

```diff
@@ -1,41 +1,46 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
eslint disable can be removed

Reply:
needed for the forEach loop

Reply:
This problem can be fixed, instead of using  `?` wrap the forEach loop inside an if condition.